bonsai/examples/governance/.cargo/config.toml

```diff
@@ -17,7 +17,6 @@ rustflags = [
   "-Doverflowing_literals",
   "-Dpath_statements",
   "-Dpatterns_in_fns_without_body",
-  "-Dprivate_in_public",
```

Comment:
Ah, thanks for catching this!

Reply:
This was the rust toolchain was pinned, because this caused a build breakage in the newest version. I think we should try to stay up to date with the latest, but we should control that instead of having builds break suddenly.